src/components/desktop/DesktopLayout.js

```diff
@@ -7,6 +7,7 @@ import aboutRoutes from '../about/aboutRoutes'
 import authRoutes from '../auth/authRoutes'
 import connectRoutes from '../connect/connectRoutes'
 import MapPage from '../map/MapPage'
+import profileRoutes from '../profile/profileRoutes'
```

Comment:
You only included it in the desktop layout - let's also have the user pages on mobile :). If you put the new route in authRoutes (see other comment), it'll work for both layouts.